cypress/integration/report_page_test.js

```diff
@@ -0,0 +1,104 @@
+describe("Report Page Test", function(){
+    beforeEach(function(){
+        cy.searchKeyword("GST");
+        cy.get(".result").click();
+        cy.url().should("include","/report");
+    });
+
+    it("can filter the topic of a search result", function(){
+        cy.get("main").find("input").type("Publications{enter}",{delay:100});
```

Comment:
This failed 3 times in CI and the screenshot shows it failed before the report page finished loading.
Can you try this instead: `cy.get(".report-page-filter").find("input").type("Publications{enter}",{delay:100});`